server/plugin/api_test.go

```diff
@@ -885,6 +1055,10 @@ func TestHandleSubscriptionNotifications(t *testing.T) {
 				return testCase.isValidChannelID
```

Comment:
one small thing rather than mocking this func I think it checks the length of the channelID maybe equal to 25 so for your mock IDs you are giving you can give it less than 25 for invalid IDs and equal to 25 for the correct one.